RELEASES.md

```diff
@@ -375,6 +376,111 @@ See also the
 [Testing the PyTorch installation](https://rocm.docs.amd.com/projects/install-on-linux/en/develop/install/3rd-party/pytorch-install.html#testing-the-pytorch-installation)
 instructions in the AMD ROCm documentation.
 
+### Installing JAX Python packages
+
+Using the index pages [listed above](#installing-rocm-python-packages), you can
+also install `jaxlib`, `jax-rocm7-plugin`, and `jax-rocm7-pjrt`.
+
+> [!NOTE]
+> By default, pip will install the latest versions of each package. If you want to
+> install older versions take note of the compatibility matrix:
+>
+> | jaxlib version | jax-rocm7-plugin version | jax-rocm7-pjrt version |
+> | ------------- | ------------------ | ------------------- |
+> | 0.7.1         | 0.7.1               | 0.7.1              |
+>
+> For example, `jax` 0.7.1 and compatible wheels can be installed by specifying
+>
+> ```
+> jax-rocm7-plugin==0.7.1 jax-rocm7-pjrt==0.7.1 jaxlib==0.7.1
+> ```
+>
+> See also
+>
+> - [Supported Jax versions in TheRock](https://github.com/ROCm/TheRock/tree/main/external-builds/jax#supported-jax-versions)
+> - [Installing previous versions of Jax](https://github.com/ROCm/rocm-jax/releases/)
+
+> [!TIP]
+> The `jax` packages depend on `rocm[libraries]`, so ROCm packages should
+> be installed First.
+> [!NOTE]
+> We will update the installation of ROCm automalically soon just by installing jax wheels.
+
+
+#### jax for gfx94X-dcgpu
+
+Supported devices in this family:
+
+| Product Name  | GFX Target |
+| ------------- | ---------- |
+| MI300A/MI300X | gfx942     |
+
+```bash
+python -m pip install \
+  --index-url https://rocm.nightlies.amd.com/v2/gfx94X-dcgpu/ \
+  --pre jax-rocm7-pjrt jax-rocm7-plugin jaxlib
+```
```

Comment:
Are there any prerelease versions on the index pages yet for JAX packages? If not, `--pre` can be omitted.